styles/_media.scss

```diff
@@ -0,0 +1,14 @@
+$mobile-width: 375px;
```

Comment:
<img width="788" alt="Screenshot 2024-05-15 at 1 44 25 PM" src="https://github.com/fpandyz/best-by-landing/assets/39587602/7522271a-a480-4685-845f-20c79ea51ef8">

We are unlikely to have any changes on 375px, so let the first breakpoint be the **tablet** one. Our task will be to make sure the website looks good even on screens smaller than 375px.